src/main/java/seedu/task/logic/parser/ParserUtil.java

```diff
@@ -72,6 +73,23 @@
         return description.isPresent() ? Optional.of(new Description(description.get())) : Optional.empty();
     }
 
+    /**
+     * Parses a {@code Optional<String> string} into an {@code Optional<String>}
+     * if {@code string} is present.
+     */
+    public static Optional<String> parseString(Optional<String> string) throws IllegalValueException {
+        assert string != null;
+        return string.isPresent() ? Optional.of(new String(string.get())) : Optional.empty();
```

Comment:
What is `Optional.of(new String(string.get()))` doing? Is this code just the `orElse` function?